onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -955,7 +993,9 @@ TEST(PoolTest, AveragePool_19_dilation_2d) {
 
   test.AddInput<float>("X", x_dims, x_vals);
   test.AddOutput<float>("Y", expected_dims, expected_vals);
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaNHWCExecutionProvider, kTensorrtExecutionProvider, kAclExecutionProvider, kOpenVINOExecutionProvider});
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "",
```

Comment:
Nit: Is this just a bad test ? Any reason why it can run on NHWC and NCHW Pool kernels ?

Reply:
Opset 19 is yet supported by the CUDA EP